doc/dev.md

```diff
@@ -197,45 +197,62 @@ cmake --build build-x64-ios
 ## iOS application (ARM-64 processors architecture)
 
 ```sh
-# Firstly, some compilation dependencies need to be installed
-brew install cmake flex bison python pkg-config autoconf automake libtool
+# Firstly, some compilation dependencies need to be installed, Homebrew could be used.
+
+/bin/bash -c "$( curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh )" # Installing Homebrew
+( echo; echo 'eval "$( /opt/homebrew/bin/brew shellenv )"' ) >> $HOME/.zprofile
+eval "$( /opt/homebrew/bin/brew shellenv )"
```

Comment:
As there are different install options, this might potentially outdate and many probably have it installed already I wonder if we should simply add a link to the website with the instructions instead of duplicating them here.